system/clusters/creodias/resource-management/rm-workspace-charts/template-hr-rm-resource-catalogue.yaml

```diff
@@ -27,3 +27,7 @@ spec:
       config:
         server:
           url: "https://resource-catalogue.{{ workspace_name }}.develop.eoepca.org"
+        # ???: do we need a `metadata` element?
```

Comment:
Sure but how would these extra metadata get filled per workspace?
Unless we just add some generic values for all workspaces.